### DISCUSSION TITLE
v0.4.0

### DISCUSSION THREAD

**Discussion by kpcyrd** [kpcyrd/narnia#9] [General]
- Update libtor from 46.6.1+0.4.6.6 to 47.7.0+0.4.7.x
- Port from structopt to clap3
- Update dependencies (including actix, nix)

## Thanks

We'd like to thank @SantiagoTorres, @repi and @rgacogne for their support on [github sponsors](https://github.com/sponsors/kpcyrd).

<hr /><em>This discussion was created from the release <a href='https://github.com/kpcyrd/narnia/releases/tag/v0.4.0'>v0.4.0</a>.</em>

**Comment by foGqiv-qykgyn-finbi1** [comment]
hacking saeid.roop
